### ISSUE TITLE
Issue in deploying the code for cdap-flume-guide

### ISSUE THREAD

**Issue by srini-daruna** [cdap-guides/cdap-flume-guide#29]
Hi,

I am using the same code as available in crap-flume-guide and I was able to successfully load the artifact. But i am observing the following error when i am trying to create an app.

Error: No application classes found in artifact 'default:crap-examples-0.0.1-SNAPSHOP'

cdap-version : 3.2.1 

Thanks,
D

**Comment by nitinmotgi** [comment]
Hi Srini, 

Did you use the release/cdap-3.2-compatible branch to build this ?

Thanks,
Nitin

**Comment by srini-daruna** [comment]
No. Just built the module using mvn clean package as per the suggestions. 

and followed the commands to artifact register and create app.
Do i need to do any other things.? i have checked in the internet about this error and could not find any resolution.

**Comment by nitinmotgi** [comment]
Can you try with the 3.2 compatible branch. There might be changes related to 3.3.

**Comment by nitinmotgi** [comment]
Srini, I was able to build the release/cdap-3.2-compatible branch and be able to deploy the application. 

```
cdap (http://Joltie-Air-2.local:10000/default)> load artifact ~/Work/Devel/cdap-flume-guide/target/
cdap-flume-guide-1.0.0.jar   classes                      generated-sources            generated-test-sources       maven-archiver               maven-status                 
surefire-reports             test-classes                 
cdap (http://Joltie-Air-2.local:10000/default)> load artifact ~/Work/Devel/cdap-flume-guide/target/cdap-flume-guide-1.0.0.jar
Successfully added artifact with name 'cdap-flume-guide'
cdap (http://Joltie-Air-2.local:10000/default)> create app WebLogAnalyticsApp cdap-flume-guide 1.0.0 user
Successfully created application
cdap (http://Joltie-Air-2.local:10000/default)> 
```

**Comment by srini-daruna** [comment]
Hi, 

I have taken the branch and built and did the same thing that you have showed. We are still getting the Error saying No Application classes found.

We are using CDAP Stand Alone VM and version is 3.2.1. 
Please help me, if i missed any thing.

**Comment by nitinmotgi** [comment]
Let me try with the VM. The behavior should be exactly the same. Will look at it and get back to you.

**Comment by nitinmotgi** [comment]
Hi Srini, 

I am able to run in from VM. Could you please attach cdap.log & cdap-debug.log) (/opt/cdap/sdk/logs/{cdap.log | cdap-debug.log }

![screen shot 2015-11-26 at 10 02 57 pm](https://cloud.githubusercontent.com/assets/1365919/11434920/1e2f7a74-9489-11e5-855b-c2e63f5d0f22.png)

Thanks,
Nitin

**Comment by srini-daruna** [comment]
Nitin, 

If we perform the maven build on my local system (Windows , running on JDK 1.8) and FTP the jar to VM, it is giving me that error. Is there any issue with that.? If i downloaded the source from github and built it in github, it is again throwing me error. Is there any issue with the Java versions or so.?

**Comment by nitinmotgi** [comment]
Srini,  Can you clone the repo cdap-flume-guide in the VM and build it there. I think VM runs 1.7 Java version. That could be a issue.
